__tests__/__fixtures__/promotionMap.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  PromotionsByTargetEnvironment,
+  EnvironmentPromotions,
+} from '../../src/promotionInfo';
+
+export const files = [
+  'teams/foundation/cron/application-values.yaml',
+  'teams/foundation/cronpgsourcer/application-values.yaml',
+  'teams/foundation/engine-graphql-schema-reporting/application-values.yaml',
+  'teams/foundation/logmetrics2datadog/application-values.yaml',
+  'teams/foundation/persistedqueries/application-values.yaml',
+  'teams/foundation/selfmonitoring/application-values.yaml',
+];
+
+export const dockerImages = [
+  'engine-cron',
+  'engine-cron',
+  'engine-graphql',
+  'logmetrics2datadog',
+  'persistedqueries',
+  'engine-self-monitoring',
+];
+
+export const commitGroups = [
+  // engine-cron commits
```

Comment:
I think it might be simpler to have an array of objects with keys `file`, `dockerImage`, `commitGroup` rather than three parallel arrays?